Makefile

```diff
@@ -82,6 +82,10 @@ CONTAINER_TOOL ?= docker
 SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
+# setting  the backend token and the backend URL 
```

Comment:
None of these settings should be required.  These can all be auto-detected by the operator at runtime.